tests/test_gui.py

```diff
@@ -46,14 +47,15 @@ def context():
     )
 
     # Make sure there is not already an automation project created
-    filters = [["name", "is", "Toolkit Publish2 UI Automation"]]
+    project_name = create_unique_name("Toolkit Publish2 UI Automation-")
```

Comment:
It would be nice if the method took care of adding the `-`. It's the little details that count!

Reply:
As a caller you shouldn't care how the suffix will be added.